analyses/molecular-subtyping-EPN/01-assign-disease-group.R

```diff
@@ -30,35 +30,56 @@ EP = pbta_histologies %>%
 
 # filter for RNA samples
 EP_rnaseq_samples = EP %>%
-  filter(experimental_strategy == "RNA-Seq") %>%
+  filter(experimental_strategy == "RNA-Seq" | experimental_strategy == "Targeted Sequencing"
+         & (!is.na(RNA_library) & experimental_strategy == "Targeted Sequencing")) %>%
   dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region) %>%
   dplyr::rename("Kids_First_Biospecimen_ID_RNA" = "Kids_First_Biospecimen_ID")
 
 # filter for DNA samples 
 WGS_dnaseqsamples = EP %>%
-  filter(experimental_strategy == "WGS") %>%
-  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region) %>%
+  filter(experimental_strategy == "WGS" | experimental_strategy == "WXS" | experimental_strategy == "Targeted Sequencing",
+         is.na(RNA_library)) %>%
+  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region, ) %>%
```

Comment:
```suggestion
  dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region) %>%
```
Everything works fine but found a typo here.